src/AutoBuild.jl

```diff
@@ -1268,79 +1278,78 @@ function build_jll_package(src_name::String, build_version::VersionNumber, code_
+        end
+        println(io, """
+                    ## Platforms
 
-              # For library products, you can use the exported variable name in `ccall()` invocations directly
-              num_chars = ccall((:count_characters, libdataproc), Cint, (Cstring, Cint), data_lines[1], length(data_lines[1]))
+                    `$(src_name)_jll.jl` is available for the following platforms:
+                    """)
+        for p in platforms
+            println(io, "* `", p, "` (`", triplet(p), "`)")
+        end
+        if length(dependencies) > 0
+            println(io)
+            println(io, """
+                        ## Dependencies
 
-              # For executable products, you can use the exported variable name as a function that you can call
-              mungify_exe() do mungify_exe_path
-                  run(`\$mungify_exe_path \$num_chars`)
-              end
-              ```
-              """)
-    end
+                        The following JLL packages are required by `$(src_name)_jll.jl`:""")
+            println(io)
+            print_jll.(Ref(io), dependencies)
+        end
+        if length(keys(products_info)) > 0
+            println(io)
+            println(io, """
+                        ## Products
 
-    # Generate LICENSE.md
-    open(joinpath(code_dir, "LICENSE.md"), "w") do io
+                        The code bindings within this package are autogenerated from the following `Products`:
+                        """)
+            for (p, _) in products_info
+                print_product(io, p)
+            end
+        end
         print(io, """
-        Copyright $(year(now()))
-
-        Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
 
-        The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+                  ## License
 
-        THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-        """)
+                  The code of this Julia library is released under the terms of the MIT "Expat" License.  This license does not apply to the binary package wrapped by the present library and whose license is shipped in the accompanying tarball.
```

Comment:
```suggestion
                  The Julia source code within this repository (all files under `src/`) are released under the terms of the MIT "Expat" License, the text of which is included below.  This license does not apply to the binary package wrapped by this Julia package and automatically downloaded by the Julia package manager upon installing this wrapper package.  The binary package's license is shipped alongside the binary itself and can be found within the `share/licenses/$name` directory within its prefix.
```

Reply:
> the text of which is included below

This is actually in the `README`.  I assume you're suggesting to move it to the `LICENSE` file, doing so.